rental.rb

```diff
@@ -0,0 +1,9 @@
+class Rental
+  attr_accessor :date, :book, :person
+
+  def initialize(date, book, person)
+    @date = date
+    @book = book
+    @person = person
+  end
```

Comment:
- [ ] When initializing the Rental class, you will need to set the "belongs-to" association between your rental and your book, and the same thing with rental and the person, so I would recommend modifying your method to something like this:
```
def initialize(date, book, person)
  @book = book
  book.rentals << self

  @person = person
  person.rentals << self
end
```
This way, you make sure that you set the proper "belongs-to" associations within rentals and book and also rentals and person.